cosmwasm/contracts/pyth/src/state.rs

```diff
@@ -44,9 +39,17 @@ pub struct PythDataSource {
 // Guardian set information
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 pub struct ConfigInfo {
-    pub owner:             Addr,
-    pub wormhole_contract: Addr,
-    pub data_sources:      HashSet<PythDataSource>,
+    pub owner:                      Addr,
+    pub wormhole_contract:          Addr,
+    pub data_sources:               HashSet<PythDataSource>,
+    pub governance_source:          PythDataSource,
+    pub governance_sequence_number: u64,
+    pub chain_id:                   u16,
+    pub valid_time_period:          Duration,
+
+    // The fee to pay, denominated in fee_denom (typically, the chain's native token)
+    pub fee:       u128,
```

Comment:
Why not storing Coin here?

Reply:
I've been copying the wormhole code for all of this stuff. Let me circle back on this (and the supporting multiple coin types) after implementing the actual payment mechanics, which will help me understand how this works.